### PR TITLE
Feat dotnet sdk

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?

- Update .NET platform and SDK generator
- Add .NET examples

## Test Plan

N/A

## Related PRs and Issues

N/A

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/config/platforms.php [hunk 1]
```diff
@@ -357,12 +357,12 @@
             [
                 'key' => 'dotnet',
                 'name' => '.NET',
-                'version' => '2.0.0',
+                'version' => '1.0.0',
```
Comment (lohanidamodar) [inline app/config/platforms.php:360]: Are we planning for 1.0? we are releasing as beta right?

**Review by lohanidamodar — COMMENTED** [review]
Looks good, just a query.

File: app/config/platforms.php (hunk 1 above)
Comment (abnegate) [reply app/config/platforms.php]: Updated to bump minor version from latest release on NuGet :+1:

**Review by lohanidamodar — COMMENTED** [review]
looks good but why are we regenerating specs for SDK release?

**Comment by abnegate** [comment]
> looks good but why are we regenerating specs for SDK release?

@lohanidamodar Just habit 😅 nothing in them had changed but there's always a diff because it's a new file. Have reverted now 👍

**Review by lohanidamodar — APPROVED** [review]

**Pull request merged**